crates/criticaltrust/src/signatures/payload.rs

```diff
@@ -327,46 +332,267 @@ mod tests {
 
     #[test]
     fn test_verify_deserialized() {
+        let root_key: PublicKey = serde_json::from_str(
+            r#"{"role":"root","algorithm":"ecdsa-p256-sha256-asn1-spki-der","expiry":null,"public":"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE4LmlE8W7eDS6WOI9Czcl+SPtoG+7SeLLCFDfYs/sP+TvOtEtYWJo8LZgI/uZu25o5qswadqPYCP3n45luTjJWg=="}"#,
```

Comment:
I've previously enjoyed using https://docs.rs/serde_json/latest/serde_json/macro.json.html which does similar to this while allowing you to escape "blob of text hell". Not sure it's appropriate for what you're testing though.

Reply:
omg this is such a life saver.